### PR TITLE
Navbar added

### PR DESCRIPTION
**Pull request by nvescio**
Summary
 - Navbar

Screenshots
Chrome
navbar

Trello Card
https://trello.com/c/8JUcMnuT/13-navbar

### CODE REVIEW

File: training_maquetado/index.html [hunk 1]
```diff
@@ -11,6 +11,11 @@
 </head>
 
 <body>
+  <div class="navbar-container">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: you can use the header tag

File: training_maquetado/scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.navbar-container {
+  background-color: white;
+  display: flex;
+  height: $nav-height;
+  box-shadow: 2px 4px 8px silver;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/navbar.scss]: use a scss color variable

File: training_maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/navbar.scss]: why the overflow?

File: training_maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  padding: 0 15%;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/navbar.scss]: use pxs

File: training_maquetado/scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  position: fixed;
+  top: 0;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/navbar.scss:10]: Is this necessary?

File: training_maquetado/index.html [hunk 6]
```diff
@@ -11,6 +11,11 @@
 </head>
 
 <body>
+  <div class="navbar-container"> 
+    <img class="logo-color" src="assets/wolox_logo.svg">
+    <h1 class="navbar-title">Developer Tools</h1>
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: Use h1 just for important or destacable content, for SEO reasons try to keep as low and descriptive as possible your h1 tags, as you already has used h1 for the titles use another one for this.

File: training_maquetado/scss/navbar.scss [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  z-index: 1;
+}
+  
+
+.logo-color {
+  height: $nav-logo-height; 
+}
+
+.navbar-title {
+  font-family: $roboto;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/navbar.scss]: I think you can define all the font-family on the styles-scss so you dont have to set it like this.

File: training_maquetado/index.html [hunk 8]
```diff
@@ -11,6 +11,11 @@
 </head>
 
 <body>
+  <header class="navbar-container"> 
+    <img class="logo-color" alt="Wolox logo" src="assets/wolox_logo.svg"/>
+    <h2 class="navbar-title">Developer Tools</h2>
+  </header>
+
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: remove line

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**